### PR TITLE
chore: add v17.x changelogs

### PR DESCRIPTION
**Pull request by MSalopek**
Closes: XXXXX

Add v17.x changelogs.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,27 @@
 # CHANGELOG
 
+## v17.0.0
+
+*2nd May, 2024*
+
+### DEPENDENCIES
+
+- Enable Partial set security - Bump [ICS](https://github.com/cosmos/interchain-security) to
```
Comment (bermuell) [inline CHANGELOG.md]: nit: suggest to use either `partial set security` or `Partial Set Security`

```suggestion
- Enable Partial Set Security - Bump [ICS](https://github.com/cosmos/interchain-security) to
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,27 @@
...
+  [v4.2.0-lsm](https://github.com/cosmos/interchain-security/releases/tag/v4.2.0-lsm)
+  ([\#3089](https://github.com/cosmos/gaia/pull/3089))
+
+### FEATURES
+
+- Enable Partial set security - Bump [ICS](https://github.com/cosmos/interchain-security) to
```
Comment (bermuell) [inline CHANGELOG.md]:
```suggestion
- Enable Partial Set Security - Bump [ICS](https://github.com/cosmos/interchain-security) to
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,27 @@
...
+  [v4.2.0-lsm](https://github.com/cosmos/interchain-security/releases/tag/v4.2.0-lsm)
+  ([\#3089](https://github.com/cosmos/gaia/pull/3089))
+
+### STATE BREAKING
+
+- Enable Partial set security - Bump [ICS](https://github.com/cosmos/interchain-security) to
```
Comment (bermuell) [inline CHANGELOG.md]:
```suggestion
- Enable Partial Set Security - Bump [ICS](https://github.com/cosmos/interchain-security) to
```

**Review by bermuell — COMMENTED** [review]
minor comments, pls address

**Review by insumity — APPROVED** [review]

**Pull request merged**